test/functional/comment_controller_test.rb

```diff
@@ -42,6 +42,29 @@ def teardown
     assert_template partial: 'questions/_comment'
   end
 
+  test 'should create wiki comments' do
+    UserSession.create(rusers(:bob))
+    assert_difference 'Comment.count' do
+      xhr :post, :create,
+          id: node(:wiki_page).nid,
+          body: 'Wiki comment'
+    end
+    assert_response :success
+    assert_not_nil :comment
+    #assert_template partial: 'wiki/_comment' 
+    #should uncomment above once comment displaying partial is implemented for wikis.
```

Comment:
I think adding regular comments to wiki pages, and making a comments tab to display them, is a good stepping stone on the way to inline comments. Then we can display all comments on the tab, with a later step of making an inline display of those comments that have reference to a specific section. (@aspriya, mayeb this is a really good step to add to your checklist!)

This may also be a great place for comments to appear by default, even if they are inline, because if we ever fail to find where they are supposed to go inline (say, if the text is edited), there's a backup plan for where they can still be found.